letsdrill.rb

```diff
@@ -1,24 +1,41 @@
 def get_letter_grade(integer)
- 
-  #Put your code here!
-
+  if integer > 89
+    'A'
+  elsif integer > 79 # Using just > here, since this elsif implies already being <= 89
+    'B'
+  elsif integer > 69 # Using just > here, since this elsif implies already being <= 79
+    'C'
+  elsif integer > 59
+    'D'
+  else
+    'F'
+  end
 end
 
 def shortest_string(array)
-
-  #Put your code here!
+  min_string = array.join
```

Comment:
@BIGred053 Take a look at the Ruby methods `min` or `min` by and see if you can simplify this answer!  Nice thought out solution though!